src/components/modal/modal.styles.js

```diff
@@ -1,32 +1,84 @@
 import { makeStyles } from '@material-ui/core/styles';
 
 export const useStyles = makeStyles((theme) => ({
+  message: {
+    textAlign: 'center',
+    fontWeight: 'bold',
+    marginBottom: 5,
+    position: 'relative',
+    marginTop: '-15px'
+    // fontSize:'5px'
+  },
+  // '@media (max-width: 950px)': {
+  //   message: {
+  //     marginTop: 3,
+  //     marginLeft: '5px',
+  //     position: 'absolute',
+  //     top: '5px',
+  //     float: 'left'
+  //   }
+  // },
   paper: {
     position: 'absolute',
-    top: '50%',
-    left: '50%',
+    top: '53%',
+    left: '51%',
+    // marginLeft:'50%',
```

Comment:
Why you commented this lines?

Reply:
I guess I have just forgot to delete this commented code.Now it is deleted